### ISSUE TITLE
no nt map file in this link

### ISSUE THREAD

**Issue by FengqingWang0712** [DaehwanKimLab/centrifuge#197]
Hi, 
When I tried to download the map file of nt database, it said that there is no this file in this link wget ftp://ftp.ncbi.nih.gov/pub/taxonomy/gi_taxid_nucl.dmp.gz

Did some one also meet this problem? How did you get the map file? Thanks for any suggestions

All the best,
Fengqing

**Comment by bioinfo17** [comment]
yes, I have the same issue - will ftp://ftp.ncbi.nih.gov/pub/taxonomy/new_taxdump/new_taxdump.tar.gz work with the current centrifuge version (v1.0.4-beta)?

**Comment by TypicalSEE** [comment]
Looks like that file is obsolete. But it still exists - https://ftp.ncbi.nih.gov/pub/taxonomy/obsolete/gi_taxid_nucl.dmp.gz

**Comment by bktorrevillas** [comment]
> Looks like that file is obsolete. But it still exists - https://ftp.ncbi.nih.gov/pub/taxonomy/obsolete/gi_taxid_nucl.dmp.gz

That whole page is now obsolete, it seems, but the centrifuge manual still points to it for building the NT database. Is there a workaround for the new NCBI formats?

**Comment by shrivastavabhishek** [comment]
Hi @bktorrevillas @bioinfo17 @TypicalSEE were you able to create the nt database, how we can choose alternative way to build the nt database if ftp://ftp.ncbi.nih.gov/pub/taxonomy/gi_taxid_nucl.dmp.gz is obsolete?

**Comment by bigben446** [comment]
the same question

**Comment by bigben446** [comment]
> > yes, I have the same issue - will ftp://ftp.ncbi.nih.gov/pub/taxonomy/new_taxdump/new_taxdump.tar.gz work with the current centrifuge version (v1.0.4-beta)?
> 
> I am not sure. I only used this way: centrifuge-download -o library -m -d "archaea,bacteria,viral,fungi" refseq > seqid2taxid.map

这么长时间过去了，你后来成功构建了centrifuge的nt新库了吗？
